website/static/js/_anonymousLogTexts.js

```diff
@@ -0,0 +1,126 @@
+var anonymousLogActions = {
+    'project_created' : 'A user created a project',
+    'project_registered' : 'A user registered a project',
+    'project_deleted' : 'A user deleted a project',
+    'created_from' : 'A user created a project based on a template',
+    'pointer_created' : 'A user created a link to a project',
+    'pointer_forked' : 'A user forked a link to a project',
+    'pointer_removed' : 'A user removed a link to a project',
+    'made_public' : 'A user made a project public',
+    'made_private' : 'A user made a project private',
+    'tag_added' : 'A user tagged a project',
+    'tag_removed' : 'A user removed a tag from a project',
+    'edit_title' : 'A user changed the title of a project',
+    'edit_description' : 'A user edited the description of a project',
+    'updated_fields' : 'A user changed the ${updated_fields} for a project',
+    'external_ids_added' : 'A user created external identifiers on a project',
+    'contributor_added' : 'A user added contributor(s) to a project',
+    'contributor_removed' : 'A user removed contributor(s) from a project',
+    'contributors_reordered' : 'A user reordered contributors for a project',
+    'permissions_update' : 'A user changed permissions for a project',
+    'made_contributor_visible' : 'A user made contributor(s) visible on a project',
+    'made_contributor_invisible' : 'A user made contributor(s) invisible on a project',
+    'wiki_updated' : 'A user updated a wiki page of a project',
+    'wiki_deleted' : 'A user deleted a wiki page of a project',
+    'wiki_renamed' : 'A user renamed a wiki page of a project',
+    'made_wiki_public' : 'A user made the wiki of a project publicly editable',
+    'made_wiki_private' : 'A user made the wiki of a project privately editable',
+    'addon_added' : 'A user added an addon to a project',
+    'addon_removed' : 'A user removed an addon from a project',
+    'addon_file_moved' : 'A user copied a file',
```

Comment:
Should read `'A user moved a file'` for consistency with non-anonymized language.